### PR TITLE
Sign up styled page

### LINKED ISSUE #19: Create Styled Sign Up Page
Create a styled sign up page based off of this Figma: https://www.figma.com/file/AU4DBu8GtFmivr0q971PpY/cityfarmslo-high-fidel?node-id=18%3A56 . You can build your work off of the existing UserSignUp component.

- [x] When styling focus more on layout and colors, don't worry about if you can't get it to perfectly match. Also, don't worry about adding any modal functionality or the image in the background
- [x] Use [styled-components](https://styled-components.com/) for CSS

### PR DESCRIPTION
**Pull request by adellevo**
- Finished styling sign up page, still having issues with eslint though 

**Notes:**
- Set min-width, so you'll need to scroll horizontally to see the full modal when the screen width is small:
Screen Shot 2022-01-23 at 5 44 19 PM
- How it looks in general:
Screen Shot 2022-01-23 at 5 44 09 PM

### CODE REVIEW

File: frontend/src/components/UserSignUp/SignUpForm.jsx [hunk 1]
```diff
@@ -3,17 +3,106 @@ import { TextField, Button } from "@material-ui/core";
...
-//  For now, console.log all of the info in a single JS object when the user clicks the submit button
+const useStyles = makeStyles(() => ({
+  root: {
+    "& .MuiFilledInput-root": {
+      background: "rgb(232, 241, 250)",
+      borderRadius: "50px",
+      borderStyle: "none",
+    },
+  },
+}));
```
Comment (vwinstea) [inline frontend/src/components/UserSignUp/SignUpForm.jsx]: What is this code doing? Is it related to the StylesProvider?

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
-        </Switch>
-      </div>
-    </Router>
+    <StylesProvider injectFirst>
+      <Router>
+        <div>
+          <Switch>
+            <Route exact path="/">
+              <Home />
+              <SignUpForm />
```
Comment (vwinstea) [inline frontend/src/App.jsx:18]: Could you move SignUpForm into a new route called "/registration"?

File: frontend/yarn.lock [hunk 3]
```diff
@@ -0,0 +1,9393 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (vwinstea) [inline frontend/yarn.lock]: Could you delete this file please? Yarn is an alternative to using npm, so its kinda redundant to have both.

File: frontend/package-lock.json [hunk 4]
```diff
@@ -1,16952 +1,8 @@
 {
```
Comment (vwinstea) [inline frontend/package-lock.json]: Thanks for reaching out about the npm version on slack! Once we get that fixed, the lockfileVersion on line 4 will hopefully go back to 2

**Review by vwinstea — CHANGES_REQUESTED** [review]
Looks really good! I left small requests/questions that I'd like to have answered before merging it in, but overall this is looking really good!

File: frontend/src/App.jsx (hunk 2 above)
Comment (adellevo) [reply frontend/src/App.jsx:18]: Yep!

File: frontend/src/components/UserSignUp/SignUpForm.jsx (hunk 1 above)
Comment (adellevo) [reply frontend/src/components/UserSignUp/SignUpForm.jsx]: It's overriding the default MUI styles for the TextField component. Just added a comment for clarity!

File: frontend/package-lock.json (hunk 4 above)
Comment (adellevo) [reply frontend/package-lock.json]: I just got my npm version to go up to 8.1.2 but now my package-lock.json file says this on line 4: "lockfileVersion": 1

Comment (vwinstea) [reply frontend/package-lock.json]: I forgot to mention you had to run `npm install` afterwards. I changed the version back this time, and in the future you should be good now that your npm version is up to date

**Review by vwinstea — APPROVED** [review]
Thanks for making those changes! Looks great!

**Pull request merged**